_quarto.yml

```diff
@@ -15,8 +14,6 @@ website:
         text: Home
       - href: getstarted.qmd
         text: Get Started
```

Comment:
I'm not sure I understand the point of this "get started" page? Is the idea to be a human curated pointer to which packages do what?

Reply:
Yes - the idea was to get the main packages on a separate page so that newcomers/starters have some sort of go-to reference.